autoload/lookup.vim

```diff
@@ -58,7 +58,13 @@ function! s:find_local_func_def(funcname) abort
     return
   endif
 
-  execute 'edit' substitute(split(funcloc, '\n')[1], '.*Last set from \ze', '', '')
```

Comment:
Since we're eventually searching for the function using `search()` anyway, we could simply skip parsing the line and simply replace this line with:

```vim
execute 'edit' matchstr(funcloc, '.*Last set from \zs.*\ze line \d\+$')
```